packages/plugins/Web3Profile/src/locales/en-US.json

```
@@ -9,6 +9,8 @@
     "NFTs": "NFTs",
     "web3_profile": "Web3 Profile",
     "EVM_wallet": "EVM Wallet",
+    "edit_profile_in_lenster": "Edit your profile in lenster",
+    "edit_profile_tips": "{{profile}} is owned by the current connected wallet. Please go to lenster.xyz to edit profile.",
```

Might contain characters that will be escaped.
```suggestion
    "edit_profile_tips": "{{- profile}} is owned by the current connected wallet. Please go to lenster.xyz to edit profile.",
```